houston/queries.go

```diff
@@ -606,4 +606,34 @@ mutation UpdateDeployment($deploymentId: Uuid!, $payload: JSON!, $executor: Exec
 			featureFlags
 		}
 	}`
+	TeamGetRequest = `
+	query GetTeam(
+		$teamUuid: Uuid
+	) {
+		team(
+			teamUuid: $teamUuid
+		)	{
+			name
+			id
+		}
+	}
+	`
+	TeamGetUsersRequest = `
+	query GetTeamUsers(
+		$teamUuid: Uuid!
+	) {
+		teamUsers(
+			teamUuid: $teamUuid
+		)	{
+			username
+			id
+			emails{
+				address
+				verified
+				primary
+			      }
```

Comment:
wrong indent

Reply:
whats the proper indention? sorry trying to fix it but all the queries look different to me and i dont think the formater is fixing it

Reply:
Yeah it's not formatted so I'm ensuring that we have some consitencies when I review this PR.

We have
```
                     emails{
				address
				verified
				primary
			      }

# should be
                     emails {
				address
				verified
				primary
                     }
```